test/restricted-post-test.js

```diff
@@ -993,9 +1013,16 @@ suite.addBatch({
                             cb = this.callback,
                             url = doc.object.likes.url;
 
-                        httputil.getJSON(url, cred, function(err, likes, response) {
-                            cb(err, likes);
-                        });
+                        Step(
+                            function(){
+                                setTimeout(this,100);
+                            },
+                            function(){
+                                httputil.getJSON(url, cred, function(err, likes, response) {
+                                    cb(err, likes);
+                                });
+                            }
+                            );
```

Comment:
Nope, nevermind. I thought that was a curly bracket.

Anyway, your `);` shouldn't be indented so far.